packages/components/src/Page/Page.tsx

```diff
@@ -32,21 +34,51 @@ interface PageProps {
    * @default standard
    */
   readonly width?: "fill" | "standard" | "narrow";
+
+  /**
+   * Page title primary action button settings.
+   */
+  readonly primaryAction?: ButtonProps;
+
+  /**
+   * Page title secondary action button settings.
+   */
+  readonly secondaryAction?: ButtonProps;
+
+  /**
+   * Page title Action menu.
+   */
+  readonly moreActionsMenu?: SectionProps[];
 }
 
 export function Page({
   title,
   intro,
   children,
   width = "standard",
+  primaryAction,
+  secondaryAction,
+  moreActionsMenu,
```

Comment:
```suggestion
  moreActionsMenu = [],
```

Reply:
Thanks, that makes so much sense. Such a clean way to handle that not being undefined. :)